src/custom/pages/App/index.tsx

```diff
@@ -11,10 +11,19 @@ import About from 'pages/About'
 import Rewards from 'pages/Rewards'
 import Faq from 'pages/Faq'
 import CowGame from 'pages/CowGame'
+import useParseReferralQueryParam from 'hooks/useParseReferralQueryParam'
 
 export const Wrapper = styled(AppMod)``
 
 export default function App() {
+  const referralAddress = useParseReferralQueryParam()
```

Comment:
For now this is fine, but I think would make sense to have an outer hook that grabs the query param and stores it on redux.
We can do it when we get to the redux task, though.

Reply:
Yes that's what I thought